From 420fcb67d3ac03fe50bd4707d4ab9fdebf87e06d Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 29 Jan 2019 12:40:21 +0000
Subject: [PATCH 350/462] MANTIS 0041015 BuildManagement - revived "resync.bat"
 so that it resyncs the globals-directories on the local-harddisk with the
 globals-directories on the network-share

git-svn-id: svn://localhost/SelfServiceBuild/trunk@631 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/resync.bat | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/scripts/resync.bat b/SelfServiceBuild/Build/scripts/resync.bat
index 2c35ad0..346a3f9 100644
--- a/SelfServiceBuild/Build/scripts/resync.bat
+++ b/SelfServiceBuild/Build/scripts/resync.bat
@@ -2,12 +2,12 @@
 rem ######################################################################
 rem #
 rem # /vob/SelfServiceBuild/Build/scripts/resync.bat
-rem # CLEARCASE ONLY - NOW OBSOLETE
+rem # now synchronising the globals-directories
 rem #
 rem ######################################################################
 
-echo Der Stream wird synchronisiert...
+echo Globals wird synchronisiert...
 
-cleartool setcs -stream
+ant -f sync-globals.build.xml
 
 echo Fertig.
-- 
2.41.0.windows.1

